### DISCUSSION TITLE
no download when using VPN

### DISCUSSION THREAD

**Discussion by oldsurferB** [transmission/transmission#6960] [Q&A]
Hi, using I mac with transmission ( 4.06)  and  using VPN , recently,  i could not download ,no peers or seeds.  unless i download without VPN ..tried several different countries all the same ? any ideas ? My VPN provider is TotalAV ...has been OK till now .  cheers

**Comment by Pentaphon** [comment]
Sounds like your VPN doesn't support port forwarding. Ask them about how they support torrent users or use a better VPN.
